tests/articles/test_comments.py

```diff
@@ -0,0 +1,123 @@
+from rest_framework import status
+from ..test_base import BaseTest
+from .test_category import TestCategory
+from rest_framework.test import APIClient
+
+
+class TestComment(BaseTest, TestCategory):
+    def setUp(self):
+        self.comment_data = {
+            "comment": {
+                "body": "His name was my name too."
+            }
+        }
+        self.second_comment_data = {
+            "comment": {
+                "body": "His name was my name too. This !"
+            }
+        }
+        self.registration_data = {
+            "user": {
+                "username": "user",
+                "email": "userstest@gmail.com",
+                "password": "Users@12345"
+            }
+        }
+        self.login_data = {
+            "user": {
+                "email": "userstest@gmail.com",
+                "password": "Users@12345"
+            }
+        }
+        self.article = {
+            "article": {
+                "title": "believer",
+                "description": "This test was created on womens day of 2019.",
+                "body": "I like to move it move it, I like to move it",
+            }
+        }
+        self.new_article = {
+            "article": {
+                "title": "believer",
+                "description": "This test was created on womens day of 2019.",
+                "body": "I like to move it move it, I like to move it",
+            }
+        }
+        self.change_comment = {
+            "comment":
+            {"body": "this is new"}
+        }
+        self.new_comment_data = {
+            "comment":
+            {"body": "this is new"}
+        }
+        self.client = APIClient()
+
+    def test_get_comment(self):
+        article_response = self.create_an_article(
+            self.article, self.registration_data)
+        slug = article_response.data["slug"]
+        comment_reponse = self.client.post(
+            "/api/articles/{}/comments/".format(str(slug)),
+            self.comment_data, format="json")
+
+        get_response = self.client.get(
+            "/api/articles/{}/comments/".format(str(slug)))
+        self.assertEqual(comment_reponse.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(get_response.status_code, status.HTTP_200_OK)
+        self.assertIn("body", get_response.data["Comment"][0])
+        # for item in get_response.data:
+        #     self.assertIn("body", item)
+        comment_id = comment_reponse.data["id"]
+
+        get_one_article = self.client.get(
+            "/api/articles/{}/comments/{}/".format(str(slug), comment_id))
+        self.assertEqual(get_one_article.data['id'], comment_id)
+        self.assertEqual(get_one_article.status_code, status.HTTP_200_OK)
+
+    def test_get_mulitple_comments_one_one_slug(self):
+        article_response = self.create_an_article(
+            self.article, self.registration_data)
+        slug = article_response.data["slug"]
+        self.client.post(
```

Comment:
@nyabongoedgar , create a method in the base_test to help you create comments. Your tests are being cluttered with posting comments.